src/tactics/LocalTransport.js

```diff
@@ -148,28 +148,29 @@ export default class LocalTransport {
    * Other Private Methods
    */
   _startSync(gameData) {
-    this._data = gameData;
+    this._data = {};
+    this._data.state = gameData;
 
     if (!gameData.ended)
       this
-        .on('startGame', ({data}) => {
-          this._data = data;
+        .on('startGame', ({data:stateData}) => {
+          this._data.state = stateData;
         })
-        .on('startTurn', ({turnId, teamId}) => {
-          Object.assign(this._data, {
-            currentTurnId: turnId,
-            currentTeamId: teamId,
+        .on('startTurn', ({ data }) => {
+          Object.assign(this._data.state, {
+            currentTurnId: data.turnId,
+            currentTeamId: data.teamId,
             actions:       [],
           });
         })
-        .on('action', ({actions}) => {
-          this._data.actions.push(...actions);
+        .on('action', ({data:actions}) => {
+          this._data.state.actions.push(...actions);
         })
-        .on('reset', ({turnId, teamId, actions}) => {
-          Object.assign(this._data, {
-            currentTurnId: turnId,
-            currentTeamId: teamId,
-            actions:       actions,
+        .on('reset', ({data}) => {
```

Comment:
@latin-programmer The 'reset' event no longer exists.  I have renamed it to the "revert" event.